hedera-node/test-clients/src/main/java/module-info.java

```diff
@@ -94,7 +94,6 @@
     requires com.hedera.node.app.service.token.impl;
     requires com.hedera.node.app.service.token;
     requires com.swirlds.base.test.fixtures;
-    requires com.swirlds.config.extensions.test.fixtures;
```

Comment:
Since I deleted `ScheduleExecutionTest.getTestConfig` method(it was not used) we no longer use `import com.swirlds.config.extensions.test.fixtures.TestConfigBuilder;`. Thus this package not needed